### PR TITLE
Create files for Payer Data Exchange

### PR DESCRIPTION
**Pull request by CaitlinV39**
## Description
This adds files for testing the Payer Data Exchange

## Related issues
N/A

## Testing
Touchstone

## FHIR Team Checklist
- [X] **Update the title** of the PR to be succinct and less than 50 characters
- [X] **Add a milestone** to the PR for the sprint that it is merged (i.e. add S47)
- [X] Tag the PR with the type of update: **Bug**, **Dependencies**, **Enhancement**, or **New-Feature**
- [X] Tag the PR with **Azure API for FHIR** if this will release to the managed service
- Review [squash-merge requirements](https://github.com/microsoft/fhir-server/blob/master/SquashMergeRequirements.md)

### Semver Change ([docs](https://github.com/microsoft/fhir-server/blob/master/docs/Versioning.md))
Skip

### CODE REVIEW

**Comment by LTA-Thinking** [comment]
Would it be better to fetch the US core definitions instead of saving copies of them to our repo? That way we don't need to worry if they update the definitions. Something like this could do it:
```
### Get the definition
# @name allergybody
GET http://hl7.org/fhir/us/core/StructureDefinition-us-core-allergyintolerance.json

### Save the definition
POST {{baseUrl}}/StructureDefinition
content-type: {{contentType}}

{{allergybody.response.body.*}}
```

File: docs/rest/PayerDataExchange/PDex_Sample_Data.http [hunk 1]
```diff
@@ -0,0 +1,284 @@
+# This file includes all the search parameters, reindex operation, and profiles to load to get the necessary pieces for the Carin Blue Button IG
```
Comment (moiradillon12) [inline docs/rest/PayerDataExchange/PDex_Sample_Data.http:1]: Update to DaVinci Payer Data Exchange

File: docs/rest/PayerDataExchange/USCore.http [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+### REST Client
+@fhirurl= <FHIR URL>
+@clientid= <CLIENT ID>
+@clientsecret= <CLIENT SECRET>
+@tenantid= <SUBSCRIPTION ID>
+@contentType = application/json
+
+### Get Metadata - Allows you to get the current capability statement
+# this will update as you add profiles to the database
+GET https://caitlinjune.azurewebsites.net/Patient/57dfd262-a0b7-4901-84d2-4909195d9022/$everything
```
Comment (moiradillon12) [inline docs/rest/PayerDataExchange/USCore.http:11]: Should this be updated to not include your URL?

**Review by moiradillon12 — COMMENTED** [review]
I tested all of the http files and was able to load required profiles, search parameters and data. Didn't test via Touchstone (limited number of tests available)

**Comment by moiradillon12** [comment]
> Would it be better to fetch the US core definitions instead of saving copies of them to our repo? That way we don't need to worry if they update the definitions. Something like this could do it:
> 
> ```
> ### Get the definition
> # @name allergybody
> GET http://hl7.org/fhir/us/core/StructureDefinition-us-core-allergyintolerance.json
> 
> ### Save the definition
> POST {{baseUrl}}/StructureDefinition
> content-type: {{contentType}}
> 
> {{allergybody.response.body.*}}
> ```

**Comment by CaitlinV39** [comment]
> Would it be better to fetch the US core definitions instead of saving copies of them to our repo? That way we don't need to worry if they update the definitions. Something like this could do it:
> 
> ```
> ### Get the definition
> # @name allergybody
> GET http://hl7.org/fhir/us/core/StructureDefinition-us-core-allergyintolerance.json
> 
> ### Save the definition
> POST {{baseUrl}}/StructureDefinition
> content-type: {{contentType}}
> 
> {{allergybody.response.body.*}}
> ```

@LTA-Thinking I think this would be amazing but it doesn't consistently work because sometimes there are issues in the profiles in the text section where the json isn't valid. I can spend more time opening JIRA issues to get those resolved and then we could move the whole file into this way.

**Comment by LTA-Thinking** [comment]
> @LTA-Thinking I think this would be amazing but it doesn't consistently work because sometimes there are issues in the profiles in the text section where the json isn't valid. I can spend more time opening JIRA issues to get those resolved and then we could move the whole file into this way.

:(
Ah, ok. That is a shame. Best to keep our own records then if the official ones aren't reliable.

**Comment by CaitlinV39** [comment]
@LTA-Thinking Moira can't approve this yet since shes not in the FHIR Server team. If you get a chance could you sign off?

**Review by LTA-Thinking — APPROVED** [review]

**Pull request merged**